static/docs/commands-reference/get.md

```diff
@@ -18,19 +18,26 @@ positional arguments:
 
 ## Description
 
-DVC provides an easy way to reuse datasets, intermediate results, ML models, or
-other files and directories tracked in another DVC repository into the current
-working directory, regardless of whether it's a DVC project. The `dvc get`
-command downloads such a <abbr>data artifact</abbr>.
-
-The `url` argument specifies the address of the Git repository containing the
-external <abbr>DVC project</abbr> (both HTTP and SSH protocols supported, e.g.
-`[user@]server:project.git`). `path` is used to specify the path of the data to
-be downloaded within the repo.
+Provides an easy way to reuse datasets, intermediate results, ML models, or
+other files and directories (any <abbr>data artifact</abbr>) tracked in another
+DVC repository, by downloading them into the current working directory.
 
 Note that this command doesn't require an existing DVC project to run in. It's a
 single-purpose command that can be used out of the box after installing DVC.
 
+The `url` argument specifies the address of the Git repository containing the
+source <abbr>DVC project</abbr>. Both HTTP and SSH protocols are supported for
+online repositories (e.g. `[user@]server:project.git`). `url` can also be a
+local file system path to an "offline" repository. The `path` argument of this
+command is used to specify the path of the data to be downloaded within the
+repo.
+
+Note that the required `path` is expected to point to a file or directory
+defined in one of the [DVC-files](/doc/user-guide/dvc-file-format) in the source
```

Comment:
@shcheklein to your comment in https://github.com/iterative/dvc.org/commit/1fd6c115210ae422b71c9eba52b94e4d761ced91#r35008668,

> defined -> controlled? referenced? I don't like that it sounds here like DVC-files come first and data second, while we always should be emphasizing that DVC-files are helpers/meta for data.

Hmmm OK updated here but we use this language in a bunch of other places, let me review...

Reply:
OK, reviewed throughout in my latest commit in this PR.

Reply:
I'm not sure we need to make a strict rule or fix other places. It's just in this case when we try to emphasize and explain that it expects data reference I would be careful with wording and usage of DVC-file stuff.

Reply:
Hmm well it's not obvious to me when wording changes should or not propagate throughout all docs. (I like to do this generally, for consistency.) Do you think the default should be to avoid that? It's typically a quick "grep", but some times it can become pretty time consuming to check everything.